tests/conftest.py

```diff
@@ -160,6 +160,15 @@ def inner(**filters):
     return inner
 
 
+@pytest.fixture
+def requires_case_sensitive_fs(tmpdir):
+    try:
+        tmpdir.mkdir('test')
```

Comment:
Please rename this to something more unique since it might conflict with actual test data otherwise. 

Also I noticed the first mkdir should be outside of the try-catch.

Reply:
Good point. I have updated the commit.